src/components/DpRadioButtonGroup/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from './DpRadioButtonGroup.vue'
```

Comment:
Do we really want to name it `DpRadioButtonGroup` when we have `DpRadio`, `DpCheckbox`, and `DpCheckboxGroup`?